config/owasp/suppressions.xml

```diff
@@ -615,4 +615,11 @@
         <packageUrl regex="true">^pkg:maven/javax\.websocket/javax\.websocket\-api@.*$</packageUrl>
         <cve>CVE-2020-11050</cve>
     </suppress>
+    <suppress>
+        <notes><![CDATA[
+file name: ant*.jar: readline.jar
+]]></notes>
```

Comment:
please chnages my typo readline.jar. Please keep file name: ant*.jar

Reply:
Addressed in this commit: https://github.com/hmcts/rd-user-profile-api/pull/354/commits/a7baee52fd84e0b46e2b159a1b25d9fbbd443f1d